apps/workplace/src/tests/book/new-desk-flow/new-desk-flow-success.component.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { MatDialog } from '@angular/material/dialog';
+import { createRoutingFactory, Spectator } from '@ngneat/spectator/jest';
+import { NewDeskFlowSuccessComponent } from 'apps/workplace/src/app/book/new-desk-flow/new-desk-flow-success.component';
+
+describe('NewDeskFlowSuccessComponent', () => {
+    let spectator: Spectator<NewDeskFlowSuccessComponent>;
+    const createComponent = createRoutingFactory({
```

Comment:
Minor note but I routing isn't needed here so use `createComponentFactory` instead.
In my snippets in VSCode for generating the boilerplate for tests I mistakenly used the `createRoutingFactory` instead `createComponentFactory`. Sorry for adding a bit of extra work on your end.